src/pages/homepage/homepage.jsx

```diff
@@ -0,0 +1,15 @@
+import { Navbar } from "../../component/navbar/navbar";
+import { Footer } from "../../component/footer/footer";
+import { Carousel} from "../../component/carousel/carousel";
+import { ProductBody } from "./product-body";
```

Comment:
create `index.js` file to import components and other files.

Reply:
_Thanks will work on suggestion_